client/src/components/events/events.js

```diff
@@ -1,59 +1,56 @@
-import { Box, Button, Flex, Heading, Spacer, IconButton } from '@chakra-ui/react';
+import {
+  Box, Flex, Heading, Spacer, IconButton,
+} from '@chakra-ui/react';
 import React, { useEffect, useState } from 'react';
-import { Header } from '../header';
-import EventList from './eventList';
 import axios from 'axios';
-import {HamburgerIcon, ChatIcon, MinusIcon} from '@chakra-ui/icons';
-import { auth } from '../firebase';
+import { HamburgerIcon, ChatIcon } from '@chakra-ui/icons';
 import { useAuthState } from 'react-firebase-hooks/auth';
 import { useLocation } from 'react-router-dom';
+import { Header } from '../header';
+import EventList from './eventList';
+import { auth } from '../firebase';
 
 const Events = () => {
-  const [user, loading, error] = useAuthState(auth);
-  let [events, setEvents] = useState([]);
-  let [listStyle, setListStyle] = useState('block');
-  let {state} = useLocation();
-  let [title, setTitle] = useState('');
+  const [user] = useAuthState(auth);
+  const [events, setEvents] = useState([]);
+  const [listStyle, setListStyle] = useState('block');
+  const { state } = useLocation();
+  const [title, setTitle] = useState('');
+
+  const getEventData = (user_id) => {
```

Comment:
ESLint said to reorder this before the `useEffect`, since the `useEffect` calls this `getEventData` function